client/js/analysis.js

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
This file should be indented with 2 spaces

Reply:
This was never completed

Reply:
Please actually complete this. This file doesn't use 2-space indents and it should

Reply:
Ah my fault...I mixed up 2-space indents and 2 lines at the top of the file. My bad.

Reply:
Please remove the 2 lines at the top of the file as well :)